### PR TITLE
[ADD] add module 'account_analytic_default_account'

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: account_analytic_default_account/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
```
Comment (adrienpeiffer) [inline account_analytic_default_account/README.rst:44]: Can you replace `{project repo}` with `account-financial-tools`

File: account_analytic_default_account/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```
Comment (adrienpeiffer) [inline account_analytic_default_account/README.rst:48]: Same remark as above. And also the name of the module and version.

File: account_analytic_default_account/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    'license': 'AGPL-3',
+    'website': 'http://www.camptocamp.com',
+    'data': [
+        'views/account_analytic_default_view.xml',
+        'views/account_voucher_payment_receipt_view.xml',
+    ],
+    'installable': True,
+    'auto_install': False,
+}
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (adrienpeiffer) [inline account_analytic_default_account/__openerp__.py]: Can you remove this line ?

**Comment by adrienpeiffer** [comment]
There are some pep8 issues

**Comment by gurneyalex** [comment]
CLA ok

**Comment by adrienpeiffer** [comment]
@damdam-s Can you check why Travis failed ?

**Comment by gurneyalex** [comment]
@damdam-s it is likely another module adding a required field which is not supplied by some new tests (or the opposite). Solution for this is to split the tests in travis.yml.

**Comment by fclementic2c** [comment]
Functional tests
:+1:

**Comment by damdam-s** [comment]
this PR will be green after https://github.com/OCA/OCB/pull/306 is merged

**Comment by pedrobaeza** [comment]
This can't be applied as it will rely only on OCB, but not Odoo, so closing this.

**Pull request closed without merging**